.github/workflows/test.yml

```diff
@@ -0,0 +1,89 @@
+name: Test
+
+on: push
+
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup node and cache yarn dependencies
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.15.0'
+          cache: 'yarn'
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Run linter
+        run: yarn lint
```

Comment:
Do you have a plan to support `stylelint` in this project? Or just only `eslint`?

Reply:
@hoangmirs I plan to add later on when working with UI setting up